aspnetcore/blazor/get-started.md

```diff
@@ -15,91 +15,115 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
+   > .NET Core SDK version 3.1.201 or later is **required** to use the 3.2 Preview 3 Blazor WebAssembly template. Confirm the installed .NET Core SDK version by running `dotnet --version` in a command shell.
+
+1. Create a new project.
+
+1. Select **Blazor App**. Select **Next**.
+
+1. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
+
+1. For a Blazor WebAssembly experience (Visual Studio 16.6 Preview 2 or later), choose the **Blazor WebAssembly App** template. For a Blazor Server experience (Visual Studio 16.4 or later), choose the **Blazor Server App** template. Select **Create**.
 
-1. Follow the guidance for your choice of tooling:
+   If the Blazor WebAssembly template isn't present, reinstall the template.
 
-   # [Visual Studio](#tab/visual-studio)
+   For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
 
-   1\. Install [Visual Studio 2019 version 16.4 or later](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload.
+1. Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> to run the app.
 
-   2\. Create a new project.
+> [!NOTE]
+> If you installed the Blazor Visual Studio extension for a prior preview release of ASP.NET Core Blazor, you can uninstall the extension. Installing the Blazor templates in a command shell is now sufficient to surface the templates in Visual Studio when using a version prior to 16.6 Preview 2.
 
-   3\. Select **Blazor App**. Select **Next**.
+# [Visual Studio Code](#tab/visual-studio-code)
 
-   4\. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
+1. Install the [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
 
-   5\. For a Blazor WebAssembly experience, choose the **Blazor WebAssembly App** template. For a Blazor Server experience, choose the **Blazor Server App** template. Select **Create**. For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>. If the Blazor WebAssembly template isn't present, return to the previous step and reinstall the template.
+1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template:
 
-   6\. Press **Ctrl**+**F5** to run the app.
+   * Install the [.NET Core 3.1.201 or later (Preview) SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+   * Run the following command in a command shell. The [Microsoft.AspNetCore.Components.WebAssembly.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Templates/) package has a preview version while Blazor WebAssembly is in preview.
```

Comment:
These sub bullets don't seem to add much. Can we just remove them? The note below already clarifies which SDK version is required.

Reply:
ok ... but I'll keep the cross-link to the SDK ... I'll move it.

I'll keep the preview note because we had **multiple readers** write in that they were confused by the preview version on the package.